doc/org/STEERING-COMMITTEE.md

```diff
@@ -0,0 +1,23 @@
+# Steering Committee
+
+This document lists the members of the Organization's Steering Committee.
+Voting members may be added once approved by the Steering Committee as
+described in the [charter](./CHARTER.md). By adding your name to this list you
+are agreeing to abide by all Organization polices, including the
+[charter](./CHARTER.md), the [code of conduct](https://github.com/vispy/vispy/blob/main/CODE_OF_CONDUCT.md), the
+[trademark policy](./TRADEMARKS.md), and the
+[antitrust policy](./ANTITRUST.md). If you are serving on the Steering
+Committee because of your affiliation with another organization (designated
+below), you represent that you have authority to bind that organization to
+these policies.
+
+| **NAME**        | **Handle**  | **Affiliated Organization** |
+|-----------------|-------------|-----------------------------|
+| David Hoese     | @djhoese    | N/A                         |
+| Almar Klein     | @almarklein | N/A                         |
+| Cyrille Rossant | @rossant    | N/A                         |
+| Nicolas Rougier | @rougier    | N/A                         |
```

Comment:
@brisvag @almarklein @larsoner @kmuehlbauer @rossant @rougier I've listed me, Almar, Cyrille, and Nicolas as the members of the steering committee. The role of the committee is described in the CHARTER.md document. The only reason I didn't list everyone is that it seemed like it would make decision making harder. I don't feel that strongly about it so let me know what you think. We could always set up some kind of rotating membership of an active steering committee and inactive members. Or we could do that type of thing with the vispy project maintainers.

Thoughts?